core/src/main/scala/cats/instances/bitSet.scala

```diff
@@ -0,0 +1,13 @@
+package cats.instances
+
+import scala.collection.immutable.BitSet
+import int.catsStdShowForInt
+import cats.syntax.show._
+import cats.Show
+
+trait BitSetInstances extends cats.kernel.instances.BitSetInstances {
+  implicit def catsStdShowForBitSet: Show[BitSet] = new Show[BitSet] {
+    def show(fa: BitSet): String =
+      fa.toIterator.map(_.show).mkString("BitSet(", ", ", ")")
```

Comment:
why would we use show here vs `fa.toString`? There is nothing generic about BitSet and the current implementation is just going to be the same as `toString` no?